app/subscriber/src/features/filter-media/FilterMediaLanding.tsx

```diff
@@ -0,0 +1,146 @@
+import { useFilterOptions } from 'components/navbar';
+import { PageSection } from 'components/section';
+import React from 'react';
+import { useContent } from 'store/hooks';
+import { Col, ISourceModel, Row, Show } from 'tno-core';
+
+import { MediaFilterTypes } from './constants';
+import { FilterMedia } from './FilterMedia';
+import * as styled from './styled';
+import { alphabetArray } from './utils';
+
+/** The FilterMediaLanding Consists of the filter to narrow down the results of media, as well as the results. It is returned in a two column layout*/
+export const FilterMediaLanding: React.FC = () => {
+  const {
+    dailyPrint,
+    sources,
+    weeklyPrint,
+    cpWire,
+    talkRadio,
+    onlinePrint,
+    television,
+    newsRadio,
+  } = useFilterOptions();
+  const [
+    {
+      mediaType: { filter },
+    },
+    { findContentWithElasticsearch, storeMediaTypeFilter: storeFilter },
+  ] = useContent();
+  const [activeFilter, setActiveFilter] = React.useState<MediaFilterTypes>(
+    MediaFilterTypes.DAILY_PRINT,
+  );
+  const [activeLetter, setActiveLetter] = React.useState<string>('All');
+  const [narrowedOptions, setNarrowedOptions] = React.useState<ISourceModel[]>([]);
+  const [activeSource, setActiveSource] = React.useState<ISourceModel | null>(null);
+
+  const labels = Object.values(MediaFilterTypes);
+  const determineOptions = () => {
+    switch (activeFilter) {
+      case MediaFilterTypes.DAILY_PRINT:
+        return dailyPrint;
+      case MediaFilterTypes.WEEKLY_PRINT:
+        return weeklyPrint;
+      case MediaFilterTypes.INTERNET:
+        return onlinePrint;
+      case MediaFilterTypes.TV:
+        return television;
+      case MediaFilterTypes.TALK_RADIO:
+        return talkRadio;
+      case MediaFilterTypes.RADIO_NEWS:
+        return newsRadio;
+      case MediaFilterTypes.CP_NEWS:
+        return cpWire;
+      default:
+        return [];
+    }
+  };
+
+  React.useEffect(() => {
+    if (activeFilter === MediaFilterTypes.WEEKLY_PRINT) {
```

Comment:
It might be better to map the use of this to a magic number of options under the section to determine whether to use a letter or the magic `All` option.  Arguably the dictionary of letters is more useful when there's more than can be seen in one block of content.  In your screen grab that would be around 18.